bedrock/mozorg/util.py

```diff
@@ -51,8 +51,14 @@ def page(name, tmpl, decorators=None, url_name=None, **kwargs):
         be applied to the view.
     @param url_name: The value to use as the URL name, default is to coerce
         the template path into a name as described above.
+    @param active_locales: A list of locale codes that should be active for this page
+        regardless of the state of the lang files. Useful for pages with locale-
+        specific templates or non-English text in the template. Ignores the lang
+        file activation tags.
+    @param add_active_locales: A list of locale codes that should be active for this page
```

Comment:
It would be nice to maybe add an example of using `add_active_locales` in the docs? I maybe can see us using this more often than `active_locales`, especially for locale-specific pages where templates may not have their own separate view or URL definition.

For example, with pages [like this one](https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/templates/firefox/whatsnew/index.id.html), instead of doing `
{% add_lang_files "firefox/whatsnew" %}` in the template, would we just be able to do the following in the view?

```python
add_active_locales = ['id']
```

Currently if we don't do the above hack, then the page is not accessible in production when `Dev=False`. We also have a similar issue when pages are en-US-only, but we forget to add it so that other translations are not available in the language switcher in the footer.

Reply:
I was just thinking of how it works preior to this PR, but I believe that's the case yeah, e.g. https://github.com/mozilla/bedrock/pull/6049

Reply:
I do mention using `add_active_locales` in the docs and that you can use it in all the places I gave examples for `active_locales`:

https://github.com/mozilla/bedrock/pull/6035/files#diff-8c556c7af0e5b741b54a3567ed7e333cR513

You think I  should add some specific examples of `add_active_locales` as well?

Reply:
> I was just thinking of how it works preior to this PR, but I believe that's the case yeah, e.g. #6049

This one is different actually. `about-en.html` is not a locale-specific-template, it's a fully separate template. The system can't know that it's meant as a variation of a template unless it's told. So I think adding the `add_langfile` tag to that one is appropriate, but the original template name lang file should be used when using the locale-specific-template feature of bedrock.

Reply:
@alexgibson does the above make sense? If so I'd like to get this in before it gets too far out-of-date.

Reply:
@pmac sorry I messed this, you're right ans yes that makes good sense. Thumbs up from me 👍